src/utils/vault-provider.ts

```
@@ -711,10 +711,35 @@ export default class VaultProvider {
+              const runeCMCId =
+                defTokens.find(
+                  (token) =>
+                    token.chain === ChainKey.THORCHAIN &&
+                    token.ticker === TickerKey.RUNE
+                )?.cmcId || 0;
+
+              return Promise.all([
+                api.coin.values([runeCMCId], Currency.USD),
+                api.coin.getAssetPriceFromMidgard("THOR.TCY"),
+              ]).then(([data, value]) => {
+                debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)